plasma_framework/contracts/src/exits/payment/controllers/PaymentProcessInFlightExit.sol

```diff
@@ -283,13 +286,17 @@ library PaymentProcessInFlightExit {
 
             // If the input has been challenged, isInputPiggybacked() will return false
             if (token == withdrawal.token && exit.isInputPiggybacked(i)) {
-                bool successBondReturn = SafeEthTransfer.transferReturnResult(
-                    withdrawal.exitTarget, withdrawal.piggybackBondSize, self.safeGasStipend
-                );
 
-                // we do not want to block a queue if bond return is unsuccessful
-                if (!successBondReturn) {
-                    emit InFlightBondReturnFailed(withdrawal.exitTarget, withdrawal.piggybackBondSize);
+                if (withdrawal.piggybackBondSize > withdrawal.bountySize) {
```

Comment:
Should this condition be checked when setting or updating the bond/bounty size?

Reply:
same, checking if there is something to return for the bond